tests/test_metric_reporter/test_data/test_metadata_directory_empty_test_results/empty.json

```diff
@@ -0,0 +1,15 @@
+{
+    "job": {
+        "dependencies": [],
+        "job_number": 1,
+        "id": "1",
+        "started_at": "2023-01-01T00:00:00Z",
+        "name": "test-job",
+        "project_slug": "test/test-project",
+        "status": "success",
+        "type": "build",
+        "stopped_at": "2023-01-01T01:00:00Z"
+    },
+    "test_metadata": [
+    ]
+}
```

Comment:
Do we need to actually have these files? We could create them using the `tmpdir` and `tmp_path` pytest fixtures.